gr-digital/lib/fll_band_edge_cc_impl.cc

```diff
@@ -158,7 +158,7 @@ namespace gr {
       int M = rint(filter_size / samps_per_sym);
       float power = 0;
 
-      // Create the baseband filter by adding two sincs together
+      // Create the baseband filter by adding two syncs together
```

Comment:
is this correct?

Reply:
actually it should be sinc, as in the sinc function

Reply:
9586034